src/app/store/authors/actions/authors.actions.ts

```diff
@@ -0,0 +1,7 @@
+import { createAction, props } from '@ngrx/store';
+
+import { Author } from '@app/interfaces/entities/author.interface';
+
+export const requestGetAuthors = createAction('[Authors] Get request');
```

Comment:
`[Authors] Get all request`, `[Authors] Get all request success`, and so on. What do you think?

Reply:
Yes, such actions names will be more informative